apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Methods/ModifyDatasetLabels.xml

```diff
@@ -0,0 +1,26 @@
+﻿<Method Name="ModifyDatasetLabels"
+        TargetType="Dataset"
+        ReturnType="IDictionary&lt;string, string&gt;">
+  <Options Type="ModifyLabelsOptions" />
+  
+  <AdditionalParameters>
+    <Parameter Name="labels" Type="IDictionary&lt;string, string&gt;" Comment="The labels to set. Must contain at least one entry; keys must not be null, but values may be (indicating label removal)." />
+  </AdditionalParameters>
+
+  <Comments>
+    <summary>
+      Sets or removes one or more labels on {target}.
+    </summary>
+    <remarks>
+      Each entry in <paramref name="labels"/> is treated as a label to set or remove. If the value is null,
+      it will be removed from the bucket; otherwise, it will be set/added. Labels which do not have an entry
```

Comment:
Whoops - yes indeed. At which point, the beauty of the system is that I hardly need to change anything :)